plugins/SitesManager/templates/_trackingCodeEmail.twig

```diff
@@ -13,12 +13,18 @@
 {% endif %}
 
 {% if ga4Used %}
-{{ 'SitesManager_GA4DetectedEmail'|translate('Google Analytics 4', 'GA', 'https://matomo.org/faq/how-to/migrate-from-google-analytics-4-to-matomo/')|raw }}
+{{ 'SitesManager_GADetectedEmail'|translate('Google Analytics 4', 'GA', 'https://matomo.org/faq/how-to/migrate-from-google-analytics-4-to-matomo/')|raw }}
 {{ 'CoreAdminHome_JSTracking_ConsentManagerEmailNote'|translate }}
 {% endif %}
 
 {% if gtmUsed %}
-{{ 'SitesManager_GTMDetectedEmail'|translate('Google Tag Manager', 'GTM', 'https://matomo.org/faq/tag-manager/migrating-from-google-tag-manager/')|raw }}
+{{ 'SitesManager_GTMDetectedEmail'|translate('https://matomo.org/faq/tag-manager/migrating-from-google-tag-manager/')|raw }}
+{{ 'CoreAdminHome_JSTracking_ConsentManagerEmailNote'|translate }}
+{% endif %}
+
+
+{% if cloudflare %}
+{{ 'SitesManager_CloudflareDetectedEmail'|translate('https://matomo.org/faq/new-to-piwik/how-do-i-install-the-matomo-tracking-code-on-my-cloudflare-setup/')|raw }}
 {{ 'CoreAdminHome_JSTracking_ConsentManagerEmailNote'|translate }}
 {% endif %}
```

Comment:
This might actually look a bit weird in the email, as if multiple stuff is detected each will be displayed with the sentence `Or you could directly embed the tracking code in your website instead:` in the end.
But I guess improving this generated email (content) might be part of another issue maybe?
ping @Stan-vw

Reply:
@Stan-vw This is what Stefan meant
![Screenshot from 2023-04-12 13-58-59](https://user-images.githubusercontent.com/14902277/231399626-7636cc8a-1a3b-4c01-bd13-abc5ca6b644a.png)

@sgiehl Updated the logic
![Screenshot from 2023-04-12 14-09-34](https://user-images.githubusercontent.com/14902277/231402734-0a3030cf-3fae-48fc-91df-f77799443bd0.png)

Reply:
I'd love to improve this email but since we want to ship these changes as quickly as possible I'm ok with the last email format.
@AltamashShaikh, give me a call when you're up, would be good for me to understand how we can iterate this email 👍